### PR TITLE
delete duplicated maven parameters

### PR DESCRIPTION
**Pull request by ilya-varchenya**

### CODE REVIEW

**Comment by ilya-varchenya** [comment]
This parameters where deleted:
-Ds3_save_screenshots=${Configuration.get(Configuration.Parameter.S3_SAVE_SCREENSHOTS)} \
-Dcore_log_level=${Configuration.get(Configuration.Parameter.CORE_LOG_LEVEL)} \
-Doptimize_video_recording=${Configuration.get(Configuration.Parameter.OPTIMIZE_VIDEO_RECORDING)} \
-Duser.timezone=${Configuration.get(Configuration.Parameter.TIMEZONE)} \

This parameter is also duplicates, but it has different values:
-Dselenium_host=${Configuration.get(Configuration.Parameter.SELENIUM_URL)} \

**Comment by ilya-varchenya** [comment]
-Dcore_log_level should be deleted from zafiraFields instead of code, that allows to override it correctly

File: src/com/qaprosoft/jenkins/pipeline/runner/maven/QARunner.groovy [hunk 1]
```diff
@@ -805,23 +804,63 @@ public class QARunner extends AbstractRunner {
...
+        goals += excludeMVNGoals(Configuration.getVars())
+        goals += excludeMVNGoals(Configuration.getParams())
 
         goals += getOptionalCapability(Configuration.Parameter.JACOCO_ENABLE, " jacoco:instrument ")
         goals += getOptionalCapability("deploy_to_local_repo", " install")
 
         logger.debug("goals: ${goals}")
         return goals
     }
+    protected def excludeMVNGoals(params) {
```
Comment (vdelendik) [inline src/com/qaprosoft/jenkins/pipeline/runner/maven/QARunner.groovy]: please rename according to groovy notation. Moreover "exclude" in name for a method which add params and vars is not a good choice... Try to find better name

File: src/com/qaprosoft/jenkins/pipeline/runner/maven/QARunner.groovy [hunk 2]
```diff
@@ -805,23 +804,63 @@ public class QARunner extends AbstractRunner {
...
+        def necessaryMavenParams  = [
```
Comment (vdelendik) [inline src/com/qaprosoft/jenkins/pipeline/runner/maven/QARunner.groovy]: add comment with descritpion why this parameters/vars shouldn't be added into the maven goals

**Pull request merged**